### PR TITLE
Suppress vulnerabilities

### PR DESCRIPTION
**Pull request by benouaer**
### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by nitinprabhuhmcts — APPROVED** [review]

File: src/main/resources/config/application.properties [hunk 1]
```diff
@@ -55,7 +55,7 @@ idam.client.secret=${IDAM_OAUTH2_CLIENT_SECRET:QM5RQQ53LZFOSIXJ}
 idam.client.redirect_uri=${IDAM_OAUTH2_REDIRECT_URL:https://localhost:3000/authenticated}
 
 #Document store
-document_management.url=${DOCUMENT_MANAGEMENT_URL:http://dm-store:5005}
+document_management.url=${DOCUMENT_MANAGEMENT_URL:http://localhost:5005}
```
Comment (hashimalisolirius) [inline src/main/resources/config/application.properties:58]: Is this related to the suppression fix?

Comment (benouaer) [reply src/main/resources/config/application.properties:58]: no, but it's needed to do any testing with tribunals, as dm-store is not accessible otherwise

**Review by hashimalisolirius — APPROVED** [review]
